xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexTradeServiceRaw.java

```diff
@@ -261,7 +303,11 @@ public BitfinexNewOrderMultiResponse placeBitfinexOrderMulti(
         new BitfinexNewOrderMultiRequest(
             String.valueOf(exchange.getNonceFactory().createValue()), bitfinexOrders);
     BitfinexNewOrderMultiResponse response =
-        bitfinex.newOrderMulti(apiKey, payloadCreator, signatureCreator, request);
+        decorateApiCall(
+                () -> bitfinex.newOrderMulti(apiKey, payloadCreator, signatureCreator, request))
+            .withRetry(retry("trade-multiOrder"))
```

Comment:
non-idempotent requests should not have retries on them, you don't want to mistakenly double send an order